store/metric.go

```diff
@@ -26,6 +25,13 @@ type Metric interface {
 // CollectRawMetrics will gather data from various of sources that is required to
 // calculate an autonomy score
 func (m *mongoDB) CollectRawMetrics(location schema.Location) (*schema.Metric, error) {
+	now := time.Now().UTC()
+	todayStartAt := time.Date(now.Year(), now.Month(), now.Day(), 0, 0, 0, 0, time.UTC)
```

Comment:
If what you want is local today in UTC time, it is not right here.

Reply:
For example, 2020/05/20 00:00:00 taiwan now time would be 2020/05/19 16:00:00 UTC time. Therefore, the todayStartAt would be 2020/05/19 00:00:00 UTC time. I am not sure if it is what you want.

Reply:
Before all the calculations were based on UTC time. Since the time zone info is available in user's profile now, I can switch to use local time.

Reply:
It's required to provide local time to the function `CollectRawMetrics`, which will bring many unexpected changes to this PR. So I prefer to keep it like this for now.